deps.edn

```diff
@@ -30,15 +31,30 @@
                          ;; org.apache.logging.log4j/log4j-to-slf4j {:mvn/version "2.23.1"}
                          ;; org.slf4j/jul-to-slf4j {:mvn/version "2.0.16"}
                          ;; org.slf4j/jcl-over-slf4j {:mvn/version "2.0.16"}
-                         org.clojure/tools.logging {:mvn/version "1.3.0"}}}
+                         org.clojure/tools.logging {:mvn/version "1.3.0"}
+
+                         ;; For native-image, used by --features=clj_easy.graal_build_time.InitClojureClasses
+                         com.github.clj-easy/graal-build-time {:mvn/version "1.0.5"}
+
+                         ;; For showing how to make buddy work in native-image
+                         buddy/buddy-sign {:mvn/version "3.5.351"}
+                         buddy/buddy-core {:mvn/version "1.11.423"}}}
 
            :dev
            ;; Include test folder here for REPL,
            ;; using test alias with iced command would also apply main-opts
            {:extra-paths ["dev/clj" "target/dev" "test/clj"]}
 
            :repl
-           {:extra-deps {}}
+           {:extra-deps {cider/cider-nrepl          {:mvn/version "0.50.2"}
```

Comment:
I kind of disagree with including user specific REPL tools in the project configuration.

Usually, cider-nrepl middleware is added by the editor when launching the REPL process. With lein tools like hashp were easy to add to the `~/.lein/profiles.clj` but I guess it isn't as easy with `~/.clojure/deps.edn` because you'd still need to refer to a alias created there to enable those deps... hmm.

Reply:
Yeah, agree too, was just for my own convenience here.

Use specific aliases need to be activated, so if a project has say a `bb dev` task bundled for easy startup, maybe there could be options in the task to specify those aliases, or actually, I wonder if babashka supports user-specific task files, since this user-specific customization needs kind of two things 1) deps.edn 2) starting with custom arguments...